webbpsf/roman.py

```diff
@@ -11,15 +11,21 @@
 import os.path
 import poppy
 import numpy as np
-from . import webbpsf_core
-from scipy.interpolate import griddata
+
+from scipy.interpolate import griddata, RegularGridInterpolator
```

Comment:
RegularGridInterpolator is not needed anymore now that we've simplified the handling of the extrapolation case.